app/javascript/src/api/account.js

```diff
@@ -62,26 +72,12 @@ export default {
     });
     return fetchPromise;
   },
-  getLabels() {
```

Comment:
Please see these lint errors also. https://app.circleci.com/jobs/github/chatwoot/chatwoot/135/parallel-runs/0/steps/0-108

Reply:
I made the changes to prevent errors from occurring. However, not sure why the lint errors are saying `Method 'handleDelete' expected no return value` & `Method 'handleGet' expected no return value`.